removing backup_storage_hook from the doc (#1366)
vitessio/website: content/en/docs/16.0/user-guides/operating-vitess/backup-and-restore/overview.md

        </ul>
      </td>
    </tr>
    <tr>
      <td><code>backup_storage_compress</code></td>
      <td>This flag controls if the backups are compressed by the Vitess code.
        By default it is set to true. Use
        <code>--backup_storage_compress=false</code> to disable.</br>
      </td>
    </tr>
    <tr>
      <td><code>backup_storage_block_size</code></td>

>       <td>If <code>--backup_storage_compress</code> is true, <code>backup_storage_block_size</code> sets the byte size for each block while compressing (default is 250000).

Reviewer: ```suggestion
      <td>If <code>--backup_storage_compress</code> is true, <code>backup_storage_block_size</code> sets the block size in bytes to use while compressing (default is 250000).
```

      </td>
    </tr>
    <tr>
      <td><code>backup_storage_number_blocks</code></td>

>       <td>If <code>--backup_storage_compress</code> is true, <code>backup_storage_number_blocks</code> sets the number of blocks that can be processed, at once, before the writer blocks, during compression. It should be equal to the number of CPUs available for compression. (default 2)

Reviewer: at once or in parallel? Not sure I understand what this flag is doing from the description.

Author: I copied the text from help. Verified the behavior , yes it should be 'parallel.. I will fix the help output text as well..
